recipes/pytorch-ignite/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pytorch-ignite" %}
+{% set version = "0.4.7" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytorch-ignite-{{ version }}.tar.gz
+  sha256: 434ebb7ee5e3da1ec00812e4280487d23fef8cb32c6790bdf2cb43d7f26a87e7
+
+build:
+  skip: true  # [win]
+  number: 0
+#   noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - pytorch
```

Comment:
The package has some min and max requirements that you should keep: `>=1.3,<2`

Reply:
Hi @beenje 
Thank you for your suggestions, which I attempted to take into account. Please let me know if you have any further edits.